src/client/components/social-share/social-share.vue

```diff
@@ -1,88 +1,124 @@
-    </button>
-
-    <div class="social-share__links">
-      <a :href="`https://twitter.com/home?status=${url}`"
-         :class="{ 'visible' : SocialSharesAreVisible }" 
+    <a class="social-share__button" @click.prevent="showOptions" href="#share" role="button">
+      <span class="a11y-sr-only">{{ !optionsAreVisible ? $t('show_sharing_options') : $t('hide_sharing_options') }}</span>
+      <img class="social-share__icon" src="/images/share.svg">
+    </a>
+
+    <div class="social-share__links" id="share">
+      <a :href="`https://twitter.com/home?status=${encodedUrl}`"
+         :class="{ 'visible' : optionsAreVisible }" 
          class="social-share__link twitter"
          target="_blank"
          rel="noopener"
          @click="handleClick('Twitter')">
-        <img class="social-share__icon" src="data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 612 612' width='512' height='512'%3E%3Cpath d='M612 116.258c-22.525 9.981-46.694 16.75-72.088 19.772 25.929-15.527 45.777-40.155 55.184-69.411-24.322 14.379-51.169 24.82-79.775 30.48-22.907-24.437-55.49-39.658-91.63-39.658-69.334 0-125.551 56.217-125.551 125.513 0 9.828 1.109 19.427 3.251 28.606-104.326-5.24-196.835-55.223-258.75-131.174-10.823 18.51-16.98 40.078-16.98 63.101 0 43.559 22.181 81.993 55.835 104.479-20.575-.688-39.926-6.348-56.867-15.756v1.568c0 60.806 43.291 111.554 100.693 123.104-10.517 2.83-21.607 4.398-33.08 4.398-8.107 0-15.947-.803-23.634-2.333 15.985 49.907 62.336 86.199 117.253 87.194-42.947 33.654-97.099 53.655-155.916 53.655-10.134 0-20.116-.612-29.944-1.721 55.567 35.681 121.536 56.485 192.438 56.485 230.948 0 357.188-191.291 357.188-357.188l-.421-16.253c24.666-17.593 46.005-39.697 62.794-64.861z' fill='%23FFF'/%3E%3C/svg%3E">
+         <span class="a11y-sr-only">{{ $t('share_on_twitter') }}</span>
```

Comment:
How about `$t('share_on_platform', { platform: 'Twitter' })`? In the CMS we can then set a single translation: "Share on {platform}".